src/OnboardingSPA/components/CheckboxTemplate/CheckboxList/index.js

```diff
@@ -0,0 +1,50 @@
+import { CheckboxItem } from '../index';
+
+/**
+ * Checkbox List Component
+ * This returns a List of Checkbox Items to be placed dynamically on screen
+ *
+ * @param          customPluginsList.callback
+ * @param {Object} customPluginsList                   - The List to be shown with a Title, Subtitle and a Description
+ *
+ * @param          customPluginsList.selectedPlugins
+ * @param          customPluginsList.customPluginsList
+ * @return CheckboxList
+ */
+const CheckboxList = ( { callback, selectedPlugins, customPluginsList } ) => {
```

Comment:
Again this is very specific to siteFeatures when you use `selectedPlugins` and `customPluginsList`. I believe the same kind of component will be used for one of the ecommerce steps so let's please make this and the `CheckboxItem` more generic so that it can support that as well